manifests/pipecd/templates/configmap.yaml

```diff
@@ -13,6 +13,7 @@ data:
 {{- if .Values.monitoring.enabled }}
 ---
 
+{{- if .Values.monitoring.clusterStats }}
```

Comment:
Lets keep the convention of nearest divider `---` belong to the doc by move this line be for L14 and the close ( `{{ end }}` ) at line L32 to L31 👀  How do you think?

Reply:
for sure. I gonna apply it!